setup/db/db/schema-41000to41100.sql

```diff
@@ -19,6 +19,10 @@
 -- Schema upgrade from 4.10.0.0 to 4.11.0.0
 --;
 
+--CLOUDSTACK-10022: Allow domain admin to create and delete subdomains
+INSERT INTO `cloud`.`role_permissions` (`uuid`, `role_id`, `rule`, `permission`, `sort_order`) values (UUID(), 3, 'createDomain', 'ALLOW', 303) ON DUPLICATE KEY UPDATE rule=rule;
```

Comment:
@GabrielBrascher sort order might be different for each environment, please allow the roles in the API cmd classes, in the `@APICommand` annotation, there is a `authorized` field you can use.